doc/source/sdk/v2/migrating.md

```diff
@@ -0,0 +1,124 @@
+
+### How does it work?
+ZAF v1 had a strict folder structure and very limited support for external libraries and modules. In contrast, ZAF v2 allows you to structure your files and folders in whichever way you like. You are also free to include external libraries and third-party modules. In order to make this possible, ZAF v2 delegates some responsibilities to the app developer. For example, the Iframe App Template uses webpack to compile Handlebars templates, SCSS stylesheets and CommonJS modules. If you'd like to use any other technologies you're welcome to do so, but you may need to re-configure the build process yourself.
+
+### Folder structure
+
+#### dist
+The dist directory is the folder you will need to package when submitting your app to the marketplace. It is also the folder you will have to serve when using ZAT. It includes your app's manifest.json file, an assets folder with all your compiled JavaScript and CSS as well as HTML and images.
+
+#### src
+The src directory is where your raw source code lives. The Iframe App Template includes different directories for javascripts, stylesheets and templates.
+
+#### .eslintrc
+eslintrc is a configuration file for [ESLint](http://eslint.org). ESLint is a linting utility for JavaScript. For more information on how to configure ESLint see [Configuring ESLint](http://eslint.org/docs/user-guide/configuring).
+
+#### package.json
+package.json is a configuration file for [NPM](https://www.npmjs.com). NPM is a package manager for JavaScript. This file includes information about your project and its dependencies. For more information on how to configure this file see [package.json](https://docs.npmjs.com/files/package.json).
+
+#### webpack.config.js
+webpack.config.js is a configuration file for [webpack](https://webpack.github.io/). Webpack is a JavaScript module bundler. For more information about webpack and how to configure it see [What is webpack](http://webpack.github.io/docs/what-is-webpack.html).
+
+### Getting Started
+The following sections will explain the steps to migrate your existing app to work with the Iframe App Template.
+
+#### Setup
+1. Clone of fork the [Iframe App Template](https://github.com/zendesk/iframe_app_template) repo.
+2. Run `npm install`
+
+#### Running locally
+In order to run your app within Zendesk you will need the [Zendesk Apps Tools (ZAT)](https://github.com/zendesk/zendesk_apps_tools). Once you've installed ZAT you will need two separate terminal tabs, one to watch and build this project and another serve it using ZAT:
+
+- `webpack --watch`
+- `zat server --path=./dist`
+
+#### Migrating your templates
```

Comment:
RE: Migrating your templates
- Suggest adding the Windows command-line equivalents